### PR TITLE
add getrandbits entry

### LINKED ISSUE #5024: [Term Entry] Python random-module .getrandbits()
### Reason/inspiration (optional)

We would like a new term entry on the `.getrandbits()` method of the [`random-module`](https://www.codecademy.com/resources/docs/python/random-module) concept in Python. The entry should go in a new file under `docs/content/python/concepts/random-module/terms/getrandbits/getrandbits.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by cjdemille**
### Issue Solved
Closes #5024 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/random-module/terms/getrandbits/getrandbits.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Functions'
+  - 'Methods'
+  - 'Random'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+In the Python `random` module, the `.getrandbits()` method is used to generate a pseudo-random integer with a specified number of bits
```
Comment (SaviDahegaonkar) [inline content/python/concepts/random-module/terms/getrandbits/getrandbits.md:17]:
```suggestion
In the Python `random` module, the `.getrandbits()` method is used to generate a pseudo-random integer with a specified number of bits
```
Make sure that the first mention of the concept should always be in bold (**bold**).

File: content/python/concepts/random-module/terms/getrandbits/getrandbits.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Syntax
+
+```pseudo
+random.getrandbits(k)
+```
+
+Where `k` is the number of bits in the generated integer. The `k` must be a non-negative integer. The result will range from 0 —2<sup>k</sup> - 1.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/random-module/terms/getrandbits/getrandbits.md:25]:
``` suggestion
Where `k` is the number of bits in the generated integer. The `k` must be a non-negative integer. The result will range from 0 —2<sup>k</sup> - 1.
```

File: content/python/concepts/random-module/terms/getrandbits/getrandbits.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Example
+
+In the example below, `.getrandbits()` returns an integer with 256 bits: 
+
+```py 
+import random 
+
+random_bits = random.getrandbits(256)
+
+```
```
Comment (SaviDahegaonkar) [inline content/python/concepts/random-module/terms/getrandbits/getrandbits.md:36]:
```suggestion
In the example below, `.getrandbits()` returns an integer with 256 bits: 

```py 
import random 

random_bits = random.getrandbits(256)

```
```
I would suggest you add another code that properly demonstrates the use of  `getrandbits` in Python. Also, add s shell block to show the output of the code.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @cjdemille,
I must say the entry is nicely written. I have suggested some changes please do it soon to get this PR approved for the first review

Thanks,
Savi

**Comment by SaviDahegaonkar** [comment]
Hey @cjdemille ,
Have you made all the suggestion I requested you, If not then please make them asap.

Thanks,
Savi

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @cjdemille , 
I had a look at your file applied some changes and approving it. Please mention which issue does this PR closes.

Thanks,
Savi

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing @cjdemille 😄 

The entry is ready to be merged 🚀

**Pull request merged**